app/views/chapters/_chapter.html.erb

```diff
@@ -79,6 +79,20 @@
     </div>
   <% end %>
 
+  <% if @next_chapter %>
+    <script type="speculationrules">
+    {
```

Comment:
Could you indent the code inside the script block? Thanks!

Reply:
Done!